lib/widgets/tab_children_widgets/p2p_swap_tab_child.dart

```diff
@@ -0,0 +1,59 @@
+import 'package:flutter/material.dart';
+import 'package:layout/layout.dart';
+import 'package:provider/provider.dart';
+import 'package:zenon_syrius_wallet_flutter/utils/utils.dart';
+import 'package:zenon_syrius_wallet_flutter/widgets/modular_widgets/p2p_swap_widgets/p2p_swaps_card.dart';
+import 'package:zenon_syrius_wallet_flutter/widgets/modular_widgets/p2p_swap_widgets/p2p_swap_options_card.dart';
+import 'package:zenon_syrius_wallet_flutter/widgets/widgets.dart';
+
+class P2pSwapTabChild extends StatelessWidget {
+  final VoidCallback onStepperNotificationSeeMorePressed;
+
+  const P2pSwapTabChild({
+    required this.onStepperNotificationSeeMorePressed,
+    Key? key,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    // TODO: Enable P2P Swaps UI on mainnet
```

Comment:
Is this true?

Reply:
Yes the UI is intentionally disabled on mainnet for now.